package.json

```diff
@@ -45,6 +46,7 @@
     "react-icons": "^4.12.0",
     "react-select": "^5.8.0",
     "use-resize-observer": "^9.1.0",
+    "vazirmatn": "^33.0.3",
```

Comment:
I suppose this is not required anymore? Since we are just using the other package. Then better `npm uninstall --save vazirmatn` then